src/orderbook.js

```diff
@@ -72,15 +71,59 @@ function OrderBook(remote,
   // books that we must keep track of to compute autobridged offers
   this._legOneBook = null;
   this._legTwoBook = null;
+  this._gotOffersFromLegOne = false;
+  this._gotOffersFromLegTwo = false;
+  this._lastAutobridgeComputeTime = 0;
+  this._autobridgeComputeTimeout = null;
 
   this._isAutobridgeable = !this._currencyGets.is_native()
     && !this._currencyPays.is_native();
 
-  function computeAutobridgedOffersWrapper() {
+  function computeAutobridgedOffersWrapperInternal() {
     self.computeAutobridgedOffers();
+    self._lastAutobridgeComputeTime = Date.now();
     self.mergeDirectAndAutobridgedBooks();
   }
 
+  function computeAutobridgedOffersWrapper() {
+    clearTimeout(self._autobridgeComputeTimeout);
+    self._autobridgeComputeTimeout = null;
+    const passed = Date.now() - self._lastAutobridgeComputeTime;
+    if (passed >= OrderBook.UPDATES_PERIOD) {
+      computeAutobridgedOffersWrapperInternal();
+    } else {
+      self._autobridgeComputeTimeout =
+        setTimeout(_.partial(computeAutobridgedOffersWrapperInternal, true),
+          OrderBook.UPDATES_PERIOD - passed);
```

Comment:
Am I understanding this correctly, the autobridged books will be processed at an interval of not less than 15 seconds? (as currently configured)